txclib/utils.py

```diff
@@ -146,11 +146,10 @@ def _prepare_url_request(host, username, password):
     # Initialize http and https pool managers
     num_pools = 1
     managers = {}
-
     if host.lower().startswith("http://"):
         scheme = "http"
-        if "http_proxy" in os.environ:
-            proxy_url = urllib3.util.url.parse_url(os.environ["http_proxy"])
+        if os.environ.get("http_proxy"):
+            proxy_url = urllib3.util.url.parse_url(scheme + "://" + os.environ["http_proxy"])
```

Comment:
If `proxy_http` includes the scheme, e.g. `proxy_https=http://127.0.0.1`, this breaks with the following message:
```
tx ERROR: HTTP error
tx ERROR: HTTPSConnectionPool(host='www.transifex.com', port=443): Max retries exceeded with url: /api/2/project/<project_slug>/resource/<resource_slug>/stats/ (Caused by ProxyError('Cannot connect to proxy.', NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x1090a64a8>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',)))
```

because it's attempting to connect to a URL like `https://http//127.0.0.1:8080`.